includes/degree-functions.php

```diff
@@ -3,6 +3,378 @@
+					<div class="degree-hours mb-5 mt-lg-5">
+						<hr>
+						<p class="h4 text-center"><?php echo $post_meta['degree_hours']; ?> <span class="font-weight-normal">total credit hours</span></p>
+						<hr>
+					</div>
+					<?php endif; ?>
+
+					<?php echo get_degree_tuition_markup( $post_meta, 'classic' ); ?>
+
+					<?php
+					if ( isset( $post_meta['degree_sidebar_content_bottom'] ) && ! empty( $post_meta['degree_sidebar_content_bottom'] ) ) {
+						echo apply_filters( 'the_content', $post_meta['degree_sidebar_content_bottom'] );
+					}
+					?>
+				</div>
+			</div>
+		</div>
+
+		<?php
+		return ob_get_clean();
+	}
+
+
+
+/**
+ * Returns HTML for the modern degree layout.
+ *
+ * @since 3.4.0
+ * @author RJ Bruneel
+ * @param object $post WP_Post object
+ * @return string HTML markup
+ *
+ */
+function get_degree_content_modern_layout( $post ) {
+
+	echo get_degree_info_modern_layout( $post );
+	echo get_degree_description_modern_layout();
+	echo get_degree_application_deadline_modern_layout();
+	echo get_degree_course_overview_modern_layout();
+	echo get_degree_skills_career_modern_layout();
+	echo get_degree_admission_requirements_modern_layout();
+
+}
+
+
+
+/**
+ * Returns HTML for the degree info section.
+ *
+ * @since 3.4.0
+ * @author RJ Bruneel
+ * @param object $post WP_Post object
+ * @return string HTML markup
+ *
+ * // TODO Make this section dynamic
+ *
+ */
+
+ function get_degree_info_modern_layout( $post ) {
+
+	$program_type        = get_degree_program_type( $post );
+	$raw_postmeta        = get_post_meta( $post->ID );
+	$post_meta           = format_raw_postmeta( $raw_postmeta );
+	$colleges_list       = get_colleges_markup( $post->ID );
+	$departments_list    = get_departments_markup( $post->ID );
+	$industry_highlights = get_field('industry_highlights');
+	$promo_image         = get_field('promo_image');
+
+	ob_start();
+	?>
+	<div class="bg-faded">
+		<div class="container program-at-a-glance py-lg-3">
+			<div class="row my-lg-3">
+				<div class="col-lg-3 p-4 pb-3 mt-3 mt-lg-0">
+
+					<h2 class="h5 font-condensed text-uppercase">Program at a Glance</h2>
+					<dl class="mt-4 mt-lg-5">
+						<?php if ( $program_type ) : ?>
+						<dt class="h6 text-uppercase text-default">Program</dt>
+						<dd class="h6 mb-4"><?php echo $program_type->name; ?></dd>
+						<?php endif; ?>
+						<?php if ( $colleges_list ) : ?>
+						<dt class="h6 text-uppercase text-default">College(s)</dt>
+						<dd class="h6 mb-4"><?php echo $colleges_list; ?></dd>
+						<?php endif; ?>
+						<?php if ( $departments_list ) : ?>
+						<dt class="h6 text-uppercase text-default">Department(s)</dt>
+						<dd class="h6"><?php echo $departments_list; ?></dd>
+						<?php endif; ?>
+					</dl>
+				</div>
+
+				<div class="col-lg-2 text-center align-self-center">
+
+					<img class="icon-calendar img-fluid" style="max-height: 4em;" role="img"
+						src="https://www.ucf.edu/online/files/2019/12/Calendar-Icon-light-gray.svg" alt="">
```

Comment:
If we know we're going to be using these icons moving forward, let's upload them to the theme instead of hotlinking them from Online.

Reply:
This will be converted to dynamic fields in a future PR